src/hooks/useStatistics.jsx

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query';
+import { getStatistics } from '../services/StatisticsService';
+
+export function useStatistics() {
+  const { data, error, isLoading, refetch } = useQuery({
+    queryKey: ['statistics'],
+    queryFn: () => getStatistics(),
+    staleTime: 5 * 60 * 1000,
+    retry: 1,
```

Comment:
Why `1`?

Reply:
misclick, I wanted to correct the line above staleTime: 1 * 60 * 1000, from 5 to 1, but I missed. I sneezed, and accidentally my fingers wrote retry. Note, all 4 letters are next to each other. stupid keyboard